docs/src/main/asciidoc/security-cors.adoc

```diff
@@ -54,7 +54,7 @@ quarkus.http.cors.access-control-allow-credentials=true <7>
 ----
 
 <1> Enables the CORS filter.
-<2> Specifies allowed origins, including a regular expression.
+<2> Specifies allowed origins, including a regular expression. When not specified, CORS headers are not sent.
```

Comment:
```suggestion
<2> Specifies allowed origins, including a regular expression. When not specified, CORS is not permitted.
```

There is another property for controlling the extra headers, so it might be a bit confusing